src/commentbot/github_utils.jl

```diff
@@ -281,3 +281,5 @@ function create_or_find_pull_request(
 
     return pr, msg
 end
+
+mention(u::GitHub.Owner) = "$(mention(u.login))"
```

Comment:
should there be a `@` in there?

```suggestion
mention(u::GitHub.Owner) = "@$(mention(u.login))"
```

Reply:
oh wait -- it's another call to mention...